examples/pytorch/rgcn/entity_classify.py

```diff
@@ -117,8 +117,7 @@ def main(args):
         backward_time.append(t2 - t1)
         print("Epoch {:05d} | Train Forward Time(s) {:.4f} | Backward Time(s) {:.4f}".
               format(epoch, forward_time[-1], backward_time[-1]))
-        train_acc = torch.sum(logits[train_idx].argmax(dim=1) == labels[train_idx]).item() / len(train_idx)
-        val_loss = F.cross_entropy(logits[val_idx], labels[val_idx])
+        cross_entropy(logits[val_idx], labels[val_idx])
```

Comment:
`F.cross_entropy`? Also, isn't `tran_acc` required in L123?

Reply:
Fixed.